tm2/pkg/sdk/testutils/testutils.go

```diff
@@ -141,19 +141,22 @@ func NewTestTxWithSignBytes(msgs []std.Msg, privs []crypto.PrivKey, fee std.Fee,
 	return tx
 }
 
-func TestAddress(name string) crypto.Address {
+func TestAddress(t *testing.T, name string) crypto.Address {
```

Comment:
This will return `!addr.Valid()` addresses.

We should use the Bech32 protocol to ensure that the key appears as follows:

`g1<human key name><padding><bech32 hash>`. Therefore, the key would look like `g1f00bar__________________afghf`.

Reply:
This has always returned invalid addresses, since the function was written, I just simplified it

I'll rewrite the test (only place where this is even called) to use valid keys for addresses, instead of shorthands

Reply:
Removed:

[5697b09](https://github.com/gnolang/gno/pull/3999/commits/5697b095c15d34e7afa0040b514ed65e615bd1bd)